docs/hardware/binos_outdoor_air_monitor.md

```diff
@@ -0,0 +1,279 @@
+| Security                        | > Security guards/ agents<br/>> CCTV Cameras                                                                                                                                                                                                                                                    |
+| Power Access                    | The site should either be a backup generator if the frequency of power outages are high Power and there should be a power socket.<br/>For solar installation: the site should assure:<br/>> Sunshine reliability<br/>> Can solar panels be added without being shielded?                        |
+| Communication reliability       | The site of installation choice should have high signal strength for easy communication with the AirQo platform. <br/>The installation site should be away from electrical interferences such as high power lines.<br/>Also, the siteshould allow:<br/>> Network connectivity of both 2G and 5G |
+| Access to the location          | The site of choice should be accessible with approval from local authorities and there should be a well constructed access road to the site.                                                                                                                                                    |
+| Distance from pollution sinks   | The installation site should be far away from: <br/>> Trees <br/> > High walls with electric fences <br/>> Storey buildings                                                                                                                                                                     |
+| Distance from pollution sources | The installation site should be far away from: <br/>> Waste incinerators <br/>> Fire <br/>> Barbecue grill <br/>> Compost pit <br/>> Construction <br/>> Hyperlocal sources such as roads, chimney                                                                                              |
+
+### 2.1.3 Pre-Installation precautions
+
+- Check whether you have all the tools and equipment
+- Ensure there are no physical damages to the monitor.
+- Take the monitor out of the packaging
+- Set up the installation location such as a pole, wall, or mount on a flat surface
+- Make sure the installation location can allow at least 270-degree airflow around the
+  monitor inlet.
+- Check the battery specification to understand whether the monitor is solar, or mains
+  powered. Bigger battery specifications are normally reserved mainly for
+  solar-powered monitors. It is worthy to note there are no limitations in having a
+  solar-powered monitor powered by mains. This is a chosen design option by the
+  AirQo team to
+
+## 2.2 Installation setup procedure
+
+![Image]()
+![Image]()
```

Comment:
@123MwanjeMike you could help with the images too

Reply:
Sure thing @HassanBahati